src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -718,13 +718,18 @@ protected static function getBundleDefaultConfig()
                 'x-forwarded-proto',
             ],
             'csrf_protection' => [
-                'enabled' => false,
+                'enabled' => null,
+                'cookie_name' => 'csrf-token',
+                'check_header' => false,
+                'double_submit_token_ids' => Array ()
```

Comment:
suggestion: short array syntax for `double_submit_token_ids` ?

Reply:
indeed, updated thanks